src/blocks/net.rs

```diff
@@ -620,34 +620,21 @@ impl ConfigBlock for Net {
         let vpn = device.is_vpn();
         let id = Uuid::new_v4().to_simple().to_string();
 
-        // List of decprecated options
-        let old_strings = [
-            "ssid",
-            "signal_strength",
-            "signal_strength_bar",
-            "bitrate",
-            "ip",
-            "ipv6",
-            "speed_up",
-            "speed_down",
-            "graph_up",
-            "graph_down",
-        ];
-
-        // if "format" option is present it will be preferred
-        let format = if config.blocks[0].1.get("format").is_some() {
+        let (_, net_config) = config
```

Comment:
What is going on here, why not just use `block_config`?
You just need to `clone()` it.

Reply:
In previous code, we were trying to compatible with options that have been deprecated, which means, it could be:

+ using `format` option
+ using deprecated options only
+ nothing

options in `block_config` would be given default values, which means `format` would not be empty.